releasetasks/templates/firefox/release_graph.yml.tmpl

```diff
@@ -69,6 +69,13 @@ tasks:
         {{ source_tasks()|indent(4) }}
     {% endif %}
 
+    {% if partner_repacks_platforms %}
```

Comment:
I guess if you did: `if partner_repacks_platforms is defined` then you don't need to define partner_repacks_platforms=[] in each test? but meh :)